store/sqlstore/file_info_store.go

```diff
@@ -230,7 +260,8 @@ func (fs SqlFileInfoStore) GetForUser(userId string) ([]*model.FileInfo, error)
 
 	if _, err := dbmap.Select(&infos,
 		`SELECT
-				*
+				*,
+				COALESCE(Content, '') as Content
```

Comment:
Should we also use `fs.queryFields` here?

Reply:
sure :facepalm:

Reply:
Done